### PR TITLE
Fix api annotations.

### PR DESCRIPTION
**Pull request by dereuromark**
Using updated sniffer.

### CODE REVIEW

File: src/SprykerMiddleware/Zed/Process/Communication/Plugin/StreamReaderStagePlugin.php [hunk 1]
```diff
@@ -20,7 +20,15 @@ class StreamReaderStagePlugin extends AbstractPlugin implements StagePluginInter
     protected const PLUGIN_NAME = 'StreamReaderStagePlugin';
 
     /**
-     * @inheritdoc
+     * {inheritDoc}
```
Comment (tamasnyulas) [inline src/SprykerMiddleware/Zed/Process/Communication/Plugin/StreamReaderStagePlugin.php]: `{@inheritdoc}`

File: src/SprykerMiddleware/Zed/Process/Communication/Plugin/Configuration/DefaultConfigurationProfilePlugin.php [hunk 2]
```diff
@@ -17,6 +17,8 @@
 class DefaultConfigurationProfilePlugin extends AbstractPlugin implements ConfigurationProfilePluginInterface
 {
     /**
+     * @api
```
Comment (tamasnyulas) [inline src/SprykerMiddleware/Zed/Process/Communication/Plugin/Configuration/DefaultConfigurationProfilePlugin.php:20]: Do we need inheritdoc for plugins same as for facades?

Comment (dereuromark) [reply src/SprykerMiddleware/Zed/Process/Communication/Plugin/Configuration/DefaultConfigurationProfilePlugin.php:20]: We probably do at some point, yes.
Out of scope probably for the current sniffer release today.

Comment (PhilinTv) [reply src/SprykerMiddleware/Zed/Process/Communication/Plugin/Configuration/DefaultConfigurationProfilePlugin.php:20]: @dereuromark could you create an enhancement ticket for that?

Comment (dereuromark) [reply src/SprykerMiddleware/Zed/Process/Communication/Plugin/Configuration/DefaultConfigurationProfilePlugin.php:20]: https://spryker.atlassian.net/browse/TE-111

**Review by PhilinTv — APPROVED** [review]

**Pull request merged**